code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -886,6 +886,12 @@
 	obj_flags |= EMAGGED
 	locked = FALSE
 
+/obj/machinery/turretid/attack_robot(mob/user)
+	if(!ailock || IsAdminGhost(user))
```

Comment:
```suggestion
	if(!ailock)
```

`IsAdminGhost` check at `attack_ai` existed because Admin ghosts' interaction piggybacked on `attack_ai`; `attack_robot`, in contrast, really is called only by robots. You can safely ditch `IsAdminGhost` check.

Reply:
Thanks, didn't see that